lib/chef/knife/bootstrap.rb

```diff
@@ -307,10 +288,9 @@ def user_name
       end
 
       def bootstrap_template
-        # The order here is important. We want to check if we have the new Chef 12 option is set first.
-        # Knife cloud plugins unfortunately all set a default option for the :distro so it should be at
```

Comment:
Does this imply any work for the cloud plugins or is this comment just out of date?

Reply:
We do need to clean this all up in knife-cloud it appears.